hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -141,4 +147,18 @@ public void frameworkMessage(ExecutorDriver driver, byte[] msg) {
         .build());
     }
   }
+
```

Comment:
We discussed and we will add a TODO here to separate the JN and NN process launch out so we don't launch at the same time in our state machine. Meanwhile, we will have to include this check for ZKFC and NN.